typings/CameraRoll.d.ts

```diff
@@ -77,7 +81,7 @@ export interface CameraRollStatic {
   /**
    * Saves the photo or video to the camera roll or photo library.
    */
-  saveToCameraRoll: (tag: string, type?: 'photo' | 'video') => Promise<string>;
+  saveToCameraRoll(tag: string, options?: SaveToCameraRollOptions): Promise<string> 
```

Comment:
I think you can rework this to become a non-breaking change by adding the additional options parameter. I agree with you that it is slightly less aesthetically pleasing but I want to avoid introducing a breaking change for this change, it will trip up a lot of people for little gain